src/lang/en.json

```diff
@@ -894,14 +894,14 @@
     "To Phone Number": "To Phone Number",
     "gtxMessagingToHint": "International format, with leading \"+\" ({e164}, {e212} or {e214})",
     "Originator type": "Originator type",
-    "Alpha (recommended)": "Alpha (recommended)",
-    "Numeric": "Numeric",
-    "Alphanumeric string (max 11 alphanumeric characters). Recipients can not reply to the message.": "Alphanumeric string (max 11 alphanumeric characters). Recipients can not reply to the message.",
-    "Numeric value (max 15 digits) with telephone number on international format without leading 00 (example UK number 07920 110 000 should be set as 447920110000). Recipients can reply to the message.": "Numeric value (max 15 digits) with telephone number on international format without leading 00 (example UK number 07920 110 000 should be set as 447920110000). Recipients can reply to the message.",
+    "Alphanumeric (recommended)": "Alphanumeric (recommended)",
+    "Telephone number": "Telephone number",
+    "cellsyntOriginatortypeDescA": "Alphanumeric string (max 11 alphanumeric characters). Recipients can not reply to the message.",
```

Comment:
I think there was a misunderstanding here: these keys are never used above.
Also try to not be cryptic here (`descA` is not a descriptive name when somebody in the future is reading the html code above ^^)